figma-shop/src/app/components/header/header-nav/header-nav.component.html

```diff
@@ -1 +1,41 @@
-<p>header-nav works!</p>
+<div class="header-main">
+    <div class="header-search">
+        <div class="header-search__logo col-3">
+            <a [routerLink]="[ '/products' ]">
+                <img src="../../../../assets/images/Freshnesecom.png" alt="logo">
+            </a>
+        </div>
+        <div class="header-search__field col-6">
+            <input type="text" name="search">
+            <img src="../../../../assets/images/search.svg" alt="search">
+        </div>
+        <div class="header-search__shop col-3">
+            <div class="header-search__person">
+                <img src="../../../../assets/images/person.png" alt="person">
+            </div>
+            <div class="header-search__bucket">
+                <img src="../../../../assets/images/bucket.svg" alt="person">
+            </div>
+        </div>
+    </div>
+
+    <div class="header-nav">
+        <nav class="nav">
+            <ul class="list">
+                <li class="item">Electronics<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Food<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Clothes<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Skin<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Toys<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Special<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Sport<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
+                <li class="item">Books<img src="../../../../assets/images/arrow-down.svg" alt="dropdown"></li>
```

Comment:
Find the way to define path to your assets like `"assets/images/... svg"`, not like `../../../... assets ...`